src/overlays/actors/ovl_Bg_Haka_Trap/z_bg_haka_trap.h

```diff
@@ -11,7 +11,14 @@ typedef void (*BgHakaTrapActionFunc)(struct BgHakaTrap*, GlobalContext*);
 typedef struct BgHakaTrap {
     /* 0x0000 */ DynaPolyActor dyna;
     /* 0x0164 */ BgHakaTrapActionFunc actionFunc;
-    /* 0x0168 */ char unk_168[0x134];
+    /* 0x0168 */ u8 actionTimer;
```

Comment:
usually we will name timer variables just "timer" unless there are multiple and it is necessary to specify what each is for

Reply:
makes sense, fixed